build/Microsoft.DotNet.Cli.Monikers.props

```diff
@@ -1,10 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <SdkBrandName>Microsoft .NET Core 1.0.1 - SDK Preview 5</SdkBrandName>
-    <SharedFrameworkBrandName>Microsoft .NET Core 1.0.1 - Runtime</SharedFrameworkBrandName>
-    <SharedHostBrandName>Microsoft .NET Core 1.0.0 - Host</SharedHostBrandName>
-    <HostFxrBrandName>Microsoft .NET Core 1.0.0 - Host FX Resolver</HostFxrBrandName>
+    <SdkBrandName>Microsoft .NET Core 1.0.3 - SDK Preview 5</SdkBrandName>
```

Comment:
I wouldn't block on it, but I think I would
* Call the LTS ones LTS
* Call the FTS ones FTS
* Call the CLI ones CLI
* CLI == LTS today, FTS later

I think this clearly shows what is what, and what CLI is using.